### PR TITLE
doc: adds documentation for `onCopy` parameter

### PR DESCRIPTION
**Pull request by wsameer**

### CODE REVIEW

**Comment by wsameer** [comment]
@igeligel let me know if the description is good enough for you?

File: README.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ Some of the unique features this component offers include:
 | onOutOptionsUpdate | function | ✔️ | Updated with new options as the parameter when outOptions language clicked |
 | maxContentLength | number | ❌  | Value that defines the maximum number of characters allowed in the text area. |
 | maxContentLengthIndicator | Object | ❌ | An Object describing how the length indicator is shown. |
+| onCopy | function | ❌ | A function you want to be called when you've copied the content of `InOutTextarea` |
```
Comment (igeligel) [inline README.md:68]: I think it should be

```suggestion
| onCopy | function | ❌ | A function that is called when you have copied the content of `InOutTextarea`. |
```

**Review by igeligel — CHANGES_REQUESTED** [review]

**Comment by wsameer** [comment]
@igeligel done!

**Review by igeligel — APPROVED** [review]
Hey @wsameer. Thanks for your contribution 👍 Would be nice if you could leave a star for the library.
I tagged this Pull Request with the necessary hacktoberfest labels, in case you are participating there. Thanks a lot, contributors like you make Open Source awesome 💯

I will also add you to the official contributor's list!

**Comment by igeligel** [comment]
@all-contributors please add @wsameer for docs

**Comment by igeligel** [comment]
@all-contributors please add @wsameer for docs

**Comment by igeligel** [comment]
@all-contributors please add @wsameer for docs

**Pull request merged**